mobile-sdk/ios/setup-intent/swift/setup-intent.swift

```diff
@@ -0,0 +1,7 @@
+import dojo_ios_sdk_drop_in_ui
+
+let dojoUI = DojoSDKDropInUI()
+dojoUI.startSetupFlow(paymentIntentId: "<paymentIntentId>", controller: self) {
```

Comment:
`dojoUI.startSetupFlow(setupIntentId:"<setupIntentId>"`
we renamed paymentIntent to setupIntent for that call

Reply:
Right. Fixed.